### DISCUSSION TITLE
Use Plausible with WordPress & domain mapping

### DISCUSSION THREAD

**Discussion by pawelkadysz** [plausible/analytics#914] [Anything Else]
Hi,

I'm using plausible with WordPress. I recently started using [domain mapping plugin](https://pl.wordpress.org/plugins/multiple-domain-mapping-on-single-site/) to build micro landing/sales pages.

Here's how it works:
I map a domain to a page on my WordPress site, like landingpage.com maps to mydomain.com/landingpage

The plugin works great, but I'm not sure how to make Plausible work with mapped domains.

I have the default Plausible code snippet on all my WordPress pages. I did put another code snippet just for the mapped domain on the page that domain maps to. But that doesn't seem to work. Heres' what I now have: 

`<script async defer data-domain="mydomain.com" src="https://plausible.io/js/plausible.exclusions.js" data-exclude="/landingpage">`
on all pages.

`<script async defer data-domain="landingpage.com" src="https://plausible.io/js/plausible.js"></script>`
only on the mapped page.

The second script doesn't capture any data.

The first one shows / when you go to landingpage.com

Any ideas how to make that work?

Thanks and have a great day!

**Comment by metmarkosaric** [comment]
thanks @pawelkadysz! have you added landingpage.com as a website in your Plausible account?

**Reply by pawelkadysz** [reply]
I did, and it says **Waiting for first pageview on landingpage.com**

**Reply by metmarkosaric** [reply]
I see. Have you looked through this integration troubleshooting guide? https://plausible.io/docs/troubleshoot-integration

Alternatively do share what URL this is and I'll take a look
